inc/namespace.php

```diff
@@ -176,16 +176,39 @@ function update_vary_headers( array $key = null, array $data = null ) : array {
  * @return bool Whether Edge Integrations have been configured and the CDN is returning data.
  */
 function edge_integrations_enabled() : bool {
-	// Get the software-supported headers.
-	$headers = get_supported_vary_headers();
-	$enabled_headers = [];
+	$edge_enabled = false;
+	$geo = Geo\get_geo();
+	$interest = Interest\get_interest();
```

Comment:
Is it possible to use a filter to remove geo or interest such that these will not work correctly?

Reply:
You can filter both things.
https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/blob/main/docs/geo.md#pantheoneiparsed_geo_data
and 
https://github.com/pantheon-systems/edge-integrations-wordpress-sdk/blob/main/docs/interest.md#pantheoneiparsed_interest_data

That's actually how the tests are working. However, I would expect if I was filtering either of those that I would be aware that I was potentially breaking/overriding the `edge_integrations_enabled` value.